### PR TITLE
Legend text

### PR DESCRIPTION
**Pull request by peterbraden**

### CODE REVIEW

File: packages/react/src/legend/LocationTotalsLegend.tsx [hunk 1]
```diff
@@ -22,6 +22,9 @@ import Disc from './Disc';
 export interface Props {
   diff?: boolean;
   colors?: Colors | DiffColors;
+  unequalText?: string;
```
Comment (ilyabo) [inline packages/react/src/legend/LocationTotalsLegend.tsx]: The naming is confusing here. Let's make one object prop instead named `textMessages`  with child props `aboutEqual`, `moreOutgoing`, and `moreIncoming`

File: packages/react/src/legend/LocationTotalsLegend.tsx [hunk 2]
```diff
@@ -53,19 +56,19 @@ const LocationTotalsLegend = ({ diff, colors }: Props) => {
         <div style={styles.item.outer}>
           <Disc size={size} inner={pos.inner} outer={pos.inner} />
           {neg && <Disc size={size} inner={neg.inner} outer={neg.inner} />}
-          <div style={styles.item.caption}>outgoing ≅ incoming</div>
+          <div style={styles.item.caption}>{unequalText || 'outgoing ≅ incoming'}</div>
```
Comment (ilyabo) [inline packages/react/src/legend/LocationTotalsLegend.tsx]: We could then create a const `DEFAULT_TEXT_MESSAGES` with these values and then merge the prop into it: 
`const messages = { ...DEFAULT_TEXT_MESSAGES, ...textMessages }`

File: packages/react/src/legend/LocationTotalsLegend.tsx (hunk 1 above)
Comment (ilyabo) [reply packages/react/src/legend/LocationTotalsLegend.tsx]: Or leave them as separate props, just change the names, as they are wrong.

**Comment by peterbraden** [comment]
Props changed

File: examples/src/index.tsx [hunk 3]
```diff
@@ -243,7 +245,21 @@ storiesOf('FlowMapLayer', module)
...
-  );
+  )
+  .add('custom legend', () => (
+    <>
+      <LegendBox bottom={35} left={10}>
+        <DiffColorsLegend positiveText="+ diff" negativeText="- diff" />
+        <hr />
+        <LocationTotalsLegend
+          diff={true}
+          aboutEqualText="unequal"
```
Comment (ilyabo) [inline examples/src/index.tsx]: Could you change the values of the `xxText` props in the story too? This way they are confusing.

**Pull request merged**